tests/backup/backup_test.go

```diff
@@ -6308,7 +6308,9 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 			log.InfoD("Restart backup pod when backup sharing is in-progress")
 			backupPodLabel := make(map[string]string)
 			backupPodLabel["app"] = "px-backup"
-			err := DeletePodWithLabelInNamespace(backup.GetPxBackupNamespace(), backupPodLabel)
+			pxbNamespace, err := backup.GetPxBackupNamespace()
+			dash.VerifyFatal(err, nil, "Getting px-backup namespace")
+			err = DeletePodWithLabelInNamespace(pxbNamespace, backupPodLabel)
 			dash.VerifyFatal(err, nil, "Restart backup pod when backup sharing is in-progress")
```

Comment:
The function `DeletePodWithLabelInNamespace()` and error `Restart backup pod when backup sharing is in-progress` are not matching, should we have a different error message here? Like `Failed to delete pods with label someLabel in namespace someNamespace"` or something?

Reply:
Also can we use a `sched-ops` function for this https://github.com/portworx/sched-ops/blob/master/k8s/core/pods.go#L61? We don't need to re-write this function `DeletePodWithLabelInNamespace()`, this method already exists

Reply:
We should remove this function and use `sched-ops` function everywhere this func is used

Reply:
@nikolaypopov We are not actually trying to delete the pod here. We want to bounce (restart) the pod. The `sched-ops` function which you suggested has a timeout parameter. I am guessing it waits for the pod to be actually deleted and if it doesn't then it context deadline will exceed and it fails right? But here the deployment spec makes sure the pod comes back up and that is what we expect.

Reply:
Discussed with @mkoppal-px, never mind this comment